pyaedt/edb.py

```diff
@@ -662,6 +662,14 @@ def close_edb(self):
         self._db.Close()
 
         self._clean_variables()
+        props = [a for a in dir(self) if not a.startswith('__')]
+
+        for a in props:
+            try:
+                del self.__dict__[a]
+            except Exception as e:
+                pass
```

Comment:
Same as for Design.py. 
You can use the one line` self.__dict__.pop(a, None)` to remove keys if you are not sure if they exist.
Or you can use `except KeyError:` instead of `except Exception as e:`. 
Unless of course there are other reasons why it would fail.

Reply:
Got it. Txs